src/sagas/publicChannels/publicChannels.selectors.ts

```diff
@@ -1,40 +1,44 @@
-import { createSelector } from 'reselect';
 import { StoreKeys } from '../store.keys';
+import { createSelector } from 'reselect';
 import { selectReducer } from '../store.utils';
 import { publicChannelsAdapter } from './publicChannels.adapter';
 import { formatMessageDisplayDate } from '../../utils/functions/formatMessageDisplayDate/formatMessageDisplayDate';
 import { certificatesMapping } from '../users/users.selectors';
 
 export const publicChannels = createSelector(
   selectReducer(StoreKeys.PublicChannels),
-  (reducerState) =>
-    publicChannelsAdapter.getSelectors().selectAll(reducerState.channels)
+  reducerState => {
+    return publicChannelsAdapter
+      .getSelectors()
+      .selectAll(reducerState.channels);
+  },
 );
 
-// TODO: Change to general channel - created by default on community creation
 export const ZbayChannel = createSelector(
   selectReducer(StoreKeys.PublicChannels),
-  (reducerState) => {
+  reducerState => {
     const publicChannelsList = publicChannelsAdapter
       .getSelectors()
       .selectAll(reducerState.channels);
 
     return publicChannelsList.find(
-      (channel) =>
-        channel.address ===
-        'zs10zkaj29rcev9qd5xeuzck4ly5q64kzf6m6h9nfajwcvm8m2vnjmvtqgr0mzfjywswwkwke68t00'
+      channel => channel.address === 'general',
```

Comment:
I would have thought of providing `'general'` as configurable variable(e.g. env)  instead of hardcoding it